src/lib/Mdsvx.d.ts

```diff
@@ -0,0 +1,7 @@
+export type RecipeMetadata = {
+	title: string;
+	layout: string;
+	date: string;
+	children?: Array<unknown>;
```

Comment:
Could be:
```suggestion
	children?: Array<RecipeMetadata>;
```
But I'm not a big fan of recursive type